src/index.ts

```diff
@@ -96,7 +40,8 @@ export const getContract = (
   chain: SupportedChains,
   provider: ethers.providers.Web3Provider | ethers.Signer
 ) => {
-  const artifact = getArtifact(chain);
+  const artifact = getRankifyInstanceArtifact(chain);
```

Comment:
Any reason why we can't we have this?
```suggestion
  const artifact = getArtifact(chain, name);
```